docs/install/aws-deployment.md

```diff
@@ -0,0 +1,52 @@
+# Install MLRun on AWS
+For AWS users, the easiest way to install MLRun is to use a native AWS deployment. This option deploys MLRun on an AWS EKS service using a cloud formation stack.
+
+## Prerequisites
+- AWS account <br>
+  See [how to create a new AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+
```

Comment:
Also, does the user need to have docker hub account or do we use ECR by default?